awx/ui_next/src/components/FieldWithPrompt/FieldWithPrompt.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { bool, func, node, string } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Tooltip } from '@patternfly/react-core';
+import { CheckboxField } from '@components/FormField';
+import { QuestionCircleIcon as PFQuestionCircleIcon } from '@patternfly/react-icons';
+import styled from 'styled-components';
+
+const QuestionCircleIcon = styled(PFQuestionCircleIcon)`
+  margin-left: 10px;
+`;
+
+const FieldHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: var(--pf-c-form__label--PaddingBottom);
+
+  label {
+    --pf-c-form__label--PaddingBottom: 0px;
+  }
+`;
+
+const StyledCheckboxField = styled(CheckboxField)`
+  --pf-c-check__label--FontSize: var(--pf-c-form__label--FontSize);
+`;
+
+function FieldWithPrompt({
+  children,
+  fieldId,
+  i18n,
+  isRequired,
+  label,
+  promptId,
+  promptName,
+  promptValidate,
```

Comment:
The prompt is always going to be a checkbox, right? I'm not sure we need a custom validate function passed in since it's always the same.

Reply:
Yea good point - I can't think of a scenario where we'd actually need any validation on one of these prompt checkboxes.  Going to remove the prop.